transformations/index.md

```diff
@@ -377,3 +377,9 @@ everything is less clear and isolated. To save time, run multiple orchestration
 #### Dependencies
 Dependencies allow you to chain transformation steps in older projects. A given transformation is executed after all 
 required steps have been executed.
+
+## Read-only Input mapping
+
+With Read-only input-mapping feature you can access all buckets (your own or linked) in transformations. Your transformation user

```

Comment:
Zásadní je ale, že není podporované přejmenovávání bucketů a ty jsou vidět pod svými IDčky... to by tam mělo být asi zmíněno... (a v TD to bude fungovat stejně nejspíš)

Reply:
no to musime nejak jeste domyslet - nazev realneho objektu (databaze/schematu) je/muze byt pro endusera neznamy (podle backendu)

Reply:
https://keboola.slack.com/archives/C048B4D0N1Y/p1672929364335709

Reply:
pridal jsem poznamku, plus v TD transformaci je informace o tom jak se sklada nazev objektu reprezentujici bucket. Bude super, kdyz to pak doplnime do UI bucketu a jednoduse tam pak odkazu "DB name is in bucket deatil" . prozatim bych to nechal takto 

viz https://keboola.slack.com/archives/C04DGPDP0AW/p1673530958307509